pkg/policy/validate.go

```diff
@@ -34,15 +34,15 @@ func Validate(policyRaw []byte, client *dclient.Client, mock bool, openAPIContro
 	}
 
 	if len(common.PolicyHasVariables(p)) > 0 && common.PolicyHasNonAllowedVariables(p) {
-		return fmt.Errorf("policy contains unknown variables")
+		return fmt.Errorf("policy contains invalid variables")
 	}
 
 	if path, err := validateUniqueRuleName(p); err != nil {
 		return fmt.Errorf("path: spec.%s: %v", path, err)
 	}
-	if p.Spec.Background == nil || (p.Spec.Background != nil && *p.Spec.Background) {
+	if p.Spec.Background == nil || *p.Spec.Background == false {
```

Comment:
Won't this result in a panic if `background == nil` ?

Reply:
No. If `p.Spec.Background == nil` then the second part of the logical OR will not be evaluated as the result is always true.

Reply:
Ah right!

But this check `p.Spec.Background != nil && *p.Spec.Background` should be converted to `*p.Spec.Background == true` right?

Reply:
I don't see that...which line are you referring to?

For `*p.Spec.Background == true` it would require a null check before dereferencing.

Reply:
Line 43, the second condition.

Reply:
fixed!